### PR TITLE
[Localization] Many corrections of recent French translations

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes?
Corrections of French translations

## Why am I doing these changes?
Because the previous ones were either inaccurate, too litteral and/or full of typos

## What did change?
Some French translations files

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/fr/achv.ts [hunk 1]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
 
   "MONO_GEN_ONE": {
     name: "Le rival originel",
-    description: "Compléter le défi de la génération 1 uniquement.",
+    description: "Terminer un challange avec uniquement des Pokémon de 1re génération.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:175]:
```suggestion
    description: "Terminer un challenge avec uniquement des Pokémon de 1re génération.",
```

File: src/locales/fr/achv.ts [hunk 2]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   },
   "MONO_GEN_TWO": {
     name: "Entre tradition et modernité",
-    description: "Compléter le défi de la génération 2 uniquement.",
+    description: "Terminer un challange avec uniquement des Pokémon de 2e génération.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:179]:
```suggestion
    description: "Terminer un challenge avec uniquement des Pokémon de 2e génération.",
```

File: src/locales/fr/achv.ts [hunk 3]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   },
   "MONO_GEN_THREE": {
-    name: "Too much water?",
-    description: "Compléter le défi de la génération 3 uniquement.",
+    name: "Too much water ?",
+    description: "Terminer un challange avec uniquement des Pokémon de 3e génération.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:183]:
```suggestion
    description: "Terminer un challenge avec uniquement des Pokémon de 3e génération.",
```

File: src/locales/fr/achv.ts [hunk 4]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   },
   "MONO_GEN_FOUR": {
-    name: "Est-elle vraiment la plus difficile?",
-    description: "Compléter le défi de la génération 4 uniquement.",
+    name: "Réellement la plus difficile ?",
+    description: "Terminer un challange avec uniquement des Pokémon de 4e génération.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:187]:
```suggestion
    description: "Terminer un challenge avec uniquement des Pokémon de 4e génération.",
```

File: src/locales/fr/achv.ts [hunk 5]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   },
   "MONO_GEN_FIVE": {
-    name: "Tous les originaux",
-    description: "Compléter le défi de la génération 5 uniquement.",
+    name: "Recast complet",
+    description: "Terminer un challange avec uniquement des Pokémon de 5e génération.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:191]:
```suggestion
    description: "Terminer un challenge avec uniquement des Pokémon de 5e génération.",
```

File: src/locales/fr/achv.ts [hunk 6]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   },
   "MONO_GEN_SIX": {
     name: "Aristocrate",
-    description: "Compléter le défi de la génération 6 uniquement.",
+    description: "Terminer un challange avec uniquement des Pokémon de 6e génération.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:195]:
```suggestion
    description: "Terminer un challenge avec uniquement des Pokémon de 6e génération.",
```

File: src/locales/fr/achv.ts [hunk 7]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   },
   "MONO_GEN_SEVEN": {
-    name: "Seulement sur le plan technique",
-    description: "Compléter le défi de la génération 7 uniquement.",
+    name: "Seulement techniquement",
+    description: "Terminer un challange avec uniquement des Pokémon de 7e génération.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:199]:
```suggestion
    description: "Terminer un challenge avec uniquement des Pokémon de 7e génération.",
```

File: src/locales/fr/achv.ts [hunk 8]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   },
   "MONO_GEN_EIGHT": {
-    name: "Le temps d'un champion !",
-    description: "Compléter le défi de la génération 8 uniquement.",
+    name: "L’heure de gloire",
+    description: "Terminer un challange avec uniquement des Pokémon de 8e génération.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:203]:
```suggestion
    description: "Terminer un challenge avec uniquement des Pokémon de 8e génération.",
```

File: src/locales/fr/achv.ts [hunk 9]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   },
   "MONO_GEN_NINE": {
-    name: "Elle y est allée doucement avec toi",
-    description: "Compléter le défi de la génération 9 uniquement.",
+    name: "Ça va, c’était EZ",
+    description: "Terminer un challange avec uniquement des Pokémon de 9e génération.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:207]:
```suggestion
    description: "Terminer un challenge avec uniquement des Pokémon de 9e génération.",
```

File: src/locales/fr/achv.ts [hunk 10]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   },
 
   "MonoType": {
-    description: "Compléter le défi du monotype {{type}}.",
+    description: "Terminer un challange en monotype {{type}}.",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:211]:
```suggestion
    description: "Terminer un challenge en monotype {{type}}.",
```

File: src/locales/fr/challenges.ts [hunk 11]
```diff
@@ -3,7 +3,7 @@ import { SimpleTranslationEntries } from "#app/plugins/i18n";
 export const challenges: SimpleTranslationEntries = {
   "title": "Paramètres du Challenge",
   "start": "Démarrer",
-  "illegalEvolution": "{{pokemon}} s'est transformé en pokemon\ninéligible pour ce défi !",
+  "illegalEvolution": "{{pokemon}} s’est transformé en Pokémon\ninéligible pour ce challange !",
```
Comment (Dakurei) [inline src/locales/fr/challenges.ts:6]:
```suggestion
  "illegalEvolution": "{{pokemon}} s’est transformé en Pokémon\ninéligible pour ce challenge !",
```

**Review by Dakurei — CHANGES_REQUESTED** [review]
"challange" does not exist

File: src/locales/fr/achv.ts [hunk 12]
```diff
@@ -172,97 +172,97 @@ export const PGMachv: AchievementTranslationEntries = {
...
   "MONO_BUG": {
-    name: "Une chenille !!!",
+    name: "Une chenille !",
   },
   "MONO_GHOST": {
-    name: "Who you gonna call?",
+    name: "SOS Fantômes",
   },
   "MONO_STEEL": {
     name: "Acier galvanisé",
```
Comment (Dakurei) [inline src/locales/fr/achv.ts:238]:
```suggestion
    name: "De type Acier !",
```
I'd be of the same opinion as them on the Discord, to replace with this more-than-familiar reference
image

See if you want to capitalize the type itself, or if you want to lowercase it if it's not the common name of the type that's being referred to.

File: src/locales/fr/battle.ts [hunk 13]
```diff
@@ -59,7 +59,7 @@ export const battle: SimpleTranslationEntries = {
   "wildPokemonWithAffix": "{{pokemonName}} sauvage",
   "foePokemonWithAffix": "{{pokemonName}} ennemi",
   "useMove": "{{pokemonNameWithAffix}} utilise\n{{moveName}} !",
-  "drainMessage": "{{pokemonName}} a été\nvidée de son énergie !",
-  "regainHealth": "{{pokemonName}} a récupéré\ndes PV!",
-  "fainted": "{{pokemonNameWithAffix}} est tombé KO!"
+  "drainMessage": "L’énergie de {{pokemonName}}\n est drainée !",
```
Comment (Dakurei) [inline src/locales/fr/battle.ts:62]:
```suggestion
  "drainMessage": "L’énergie de {{pokemonName}}\nest drainée !",
```
No space after breaking line

**Review by Dakurei — APPROVED** [review]
Everything looks good to me this time 👍

**Comment by Dakurei** [comment]
_Well ... If I were being fussy, I'd ask you to take this opportunity to correct this 😂_
https://github.com/pagefaultgames/pokerogue/blob/cc310ffd3a96415e0cad2862f5f46acaa95babee/src/locales/fr/battle.ts#L57

With the same text that has been present since 2G across all generations
`Hein` instead of `Oh`


Pokemon-Diamant-Perle-Oeuf-Manaphy-02-1160x653
image

**Comment by Dakurei** [comment]
Perfect 👌

**Review by XGSleepWalker — APPROVED** [review]

**Pull request merged**